indigo/indigo/src/main/scala/indigo/shared/events/GlobalEvent.scala

```diff
@@ -181,6 +182,21 @@ object MouseEvent:
     def apply(x: Int, y: Int): Move =
       Move(Point(x, y))
 
+  /** The mouse wheel was rotated a certain amount into the Y axis.
+    *
+    * @param x
+    *   X coord relative to magnification level
+    * @param y
+    *   Y coord relative to magnification level
```

Comment:
These are wrong, I imagine they're a copy paste and the real person at fault is me from when I changed the `Click` event to hold a `Point`. 😄 

Also needs the amount field,

Reply:
Copy-pasted 😅